.github/workflows/build_opa_policies.yml

```diff
@@ -0,0 +1,318 @@
+---
+name: Tessell Pipelines Code Build
+
+on:
+  workflow_call:
+
+jobs:
+  build:
+    runs-on: self-hosted
+    outputs:
+      mail_id_output: ${{ steps.mail_info.outputs.mail_id }}
+      mail_subject_output: ${{ steps.mail_info.outputs.mail_subject}}
+      mail_body_output: ${{ steps.mail_info.outputs.mail_body }}
+    env:
+      SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL  }}
+      NEXUS_SERVER: nexus.tessell.cloud
+      NEXUS_ARTIFACT_REPO: tessell-artifacts
+      GITHUB_USER: ${{ secrets.CIPIPELINE_NEXUS_USERNAME }}
+      GITHUB_TOKEN: ${{ secrets.CIPIPELINE_NEXUS_PASSWORD }}
+
+    steps:
+      - name: Get branch name
+        shell: bash
+        run: |
+          echo "SOURCE_BRANCH=$(echo ${GITHUB_REF#refs/heads/} | tr / -)" >> $GITHUB_ENV
+
+      - name: Set Label
+        if: ${{ startsWith(env.SOURCE_BRANCH,'rel-') || env.SOURCE_BRANCH == 'main' }}
+        shell: bash
+        run: |
+          if [[ ${{env.SOURCE_BRANCH}} == 'main' ]]; then
+            TOKEN=$(echo -n '${{secrets.CIPIPELINE_NEXUS_USERNAME}}:${{secrets.CIPIPELINE_NEXUS_PASSWORD}}' | base64)
+            LABEL=$(curl --location 'http://api.dev.convoy.tessell.cloud/devops/applications/app-groups/tessell/latest-main-release-label' \
```

Comment:
I created 2 variables `CONVOY_API_ENDPOINT` and `CONVOY_DEV_API_ENDPOINT` ; Use them. Also in a separate PR, change in other ci files also. This will be easy to do testing and then switch back to CONVOY prod.